src/include/ftp-server/dialogs.rb

```diff
@@ -1365,7 +1365,7 @@ def addit_settings
             )
           ), # end of `CheckBoxFrame(`id("SSLEnable"), "Sec&urity Settings", true
           VSpacing(1),
-          Frame(_("SUSEfirewall Settings"), HBox(HSpacing(1), "Firewall")),
+          Left("Firewall"),
```

Comment:
_("Firewall") ???

Reply:
Don't think so, originally only _("SUSEfirewall Settings") was translated and "Firewall" is the cwm definition..

Reply:
You're right

Reply:
yes, it is ID, not label.